src/test/resources/pages/instructorCoursesAddDupIdFailed.html

```diff
@@ -112,7 +112,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
```

Comment:
This line retrieves the current date & time, thus cannot be hard-coded into HTML file. Some tests failed because of this.